model-api/src/commonTest/kotlin/org/modelix/model/api/BuiltinLanguagesTest.kt

```diff
@@ -41,4 +43,15 @@ class BuiltinLanguagesTest {
         // They were only accessible by directly calling Model.modelImports for example.
         assertEquals(3, childLinks.size)
     }
+
+    @Test
+    fun allBuildInLanguagesHaveMpsConceptId() {
```

Comment:
```suggestion
    fun allBuiltInLanguagesHaveMpsConceptId() {
```

Reply:
Maybe we should even change the format to all test methods here to our convention with backticks and spaces.

Reply:
Done.
Fixed the grammatical error in the name.